helm/charts/hydra/templates/hpa.yaml

```
@@ -14,34 +14,18 @@ spec:
   minReplicas: {{ .Values.deployment.autoscaling.minReplicas }}
   maxReplicas: {{ .Values.deployment.autoscaling.maxReplicas }}
   metrics:
-  {{- if .Values.deployment.autoscaling.targetMemoryUtilizationPercentage }}
-    - type: Resource
-      resource:
-        name: memory
-        target:
-          type: Utilization
-          averageUtilization: {{ .Values.deployment.autoscaling.targetMemoryUtilizationPercentage }}
-  {{- else if .Values.deployment.autoscaling.targetMemoryAverageValue }}
-    - type: Resource
-      resource:
-        name: memory
-        target:
-          type: AverageValue
-          averageValue: {{ .Values.deployment.autoscaling.targetMemoryAverageValue }}
+  {{- with .Values.deployment.autoscaling.targetCPU}}
+  - type: Resource
+    resource:
+      name: cpu
+      target:
+        {{ toYaml . | nindent 8 }}
```

```suggestion
        {{- toYaml . | nindent 8 }}
```
with `nindent` it is better to remove all whitespace from the left, otherwise we end up with extra whitespaces in the result https://github.com/ory/k8s/actions/runs/4135703437/jobs/7148546628#step:4:1264 :)